Automattic/woocommerce-payments
# Add handling for server export for deposits CSV download

Fixes #3847 

#### Changes proposed in this Pull Request
Trigger request to the server when requesting a download of deposits table while using filters. Otherwise, only download deposits present in the table view.









#### Testing instructions





Note this PR relies on this server PR: 1854-gh-Automattic/woocommerce-payments-server

- Via WP-Admin go to Payments > Deposits
- Click the download link.
- Ensure a CSV containing the deposits listed in the table view immediately downloads
- Select any filter
- Ensure a notice appears letting you know an export will be emailed to you
- Confirm you receive an email containing a download link for an export containing deposits that meet the criteria (look at server PR's testing instructions for reference)
- Repeat steps 3-6 with different combinations of filters

-------------------


**Post merge**
